test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test

```diff
@@ -155,7 +155,7 @@ Date: REGEX(.*)
   <statusCode>
     <code>404</code>
     <reasonPhrase>No context element found</reasonPhrase>
-    <details>Entity-Attribute pair: /E01-A1/</details>
+    <details>Entity id: /E01/</details>
```

Comment:
Good point. Along with backward compatibility is preserved (as in this case), let's deal with error message improvements in the future (in the case they are needed at the end).

NTC